ignite/engine/engine.py

```diff
@@ -334,7 +334,11 @@ def on(self, event_name, *args, **kwargs):
         """
 
         def decorator(f: Callable) -> Callable:
-            self.add_event_handler(event_name, f, *args, **kwargs)
+            if type(event_name) is EventList:
```

Comment:
Seems like EventList are only supported for decorator. It remains `engine.add_event_handler`

Reply:
Yep, I didn't realize that. I fix it.